lib/Controller.js

```diff
@@ -15,8 +16,10 @@ var Q = require('q');
 var qio = require('q-io/fs');
 var path = require('path');
 var util = require('util');
-var adapter = require('./Adapters/AdapterFactory.js').getAdapter();
+
 var MonitorServer=require('./MonitorServer.js');
+const events = require('events');
+
```

Comment:
Guy,
there is one change that is required, unless I add the line to App.js...
var adapter = require('./Adapters/AdapterFactory.js').getAdapter();
moved to the top of Controller.js so the configuration updates will take place before any module get config.

From: kalturaguy <notifications@github.com<mailto:notifications@github.com>>
Reply-To: kaltura/liveDVR <reply@reply.github.com<mailto:reply@reply.github.com>>
Date: Sunday, 4 September 2016 at 7:53 AM
To: kaltura/liveDVR <liveDVR@noreply.github.com<mailto:liveDVR@noreply.github.com>>
Cc: Lilach Maliniak <Lilach.Maliniak@kaltura.com<mailto:Lilach.Maliniak@kaltura.com>>, Author <author@noreply.github.com<mailto:author@noreply.github.com>>
Subject: Re: [kaltura/liveDVR] Regression test implementation tested for single flavor (#225)


In lib/Controller.js<https://github.com/kaltura/liveDVR/pull/225#discussion_r77446694>:

>  var MonitorServer=require('./MonitorServer.js');
> +const events = require('events');
> +


nothing in this file really change, so for each merge revert all space changes so this file will not be in the pull reuqest

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/kaltura/liveDVR/pull/225/files/e85102102edab8ddf2f5230112757ce3f27958b0#r77446694>, or mute the thread<https://github.com/notifications/unsubscribe-auth/ASW-v-HbodbwvKsTy5O1VAe53hks0E6-ks5qmk7bgaJpZM4JOb_J>.